example/http/client/async-ssl/http_client_async_ssl.cpp

```diff
@@ -216,9 +228,12 @@ int main(int argc, char** argv)
     auto const target = argv[3];
     int version = argc == 5 && !std::strcmp("1.0", argv[4]) ? 10 : 11;
 
-    // The io_context is required for all I/O
+    // The io_context is one possible executor context
     net::io_context ioc;
 
+    // All handlers will be invoked through this executor
+    auto exec = ioc.get_executor();
```

Comment:
This executor doesn't need to be named

Reply:
...but if it did _need_ a name (which it doesn't, here) then the canonical name for an object of _Executor_ type when only one such instance exists in the scope is `ex`.